app/assets/javascripts/timeout/timeoutDialog.js

```diff
@@ -271,6 +271,10 @@ function timeoutDialog(options) {
     window.location.href = settings.signOutUrl
   }
 
+  function signedYouOut() {
```

Comment:
can we modified singedOut function, passing argument to it?

Reply:
My view is that it is easier to understand, more reliable and more maintainable to have the 2 one line functions, rather than add a variable and introduce conditional logic in the function. I would prefer to keep it as is.